frontend/src/recipePage.jsx

```diff
@@ -0,0 +1,83 @@
+import './recipePage.css';
+import React, { useState } from 'react';
+
+export default function RecipePage({name, desc, image, ingredients, steps}) {
+  const [anIngredient, setNewIngredient] = React.useState('');
+  const [anInstruction, setNewInstruction] = React.useState('');
+
+  const addIngredient = () => {
+    console.log(anIngredient)
+    var ul = document.getElementById("ingredientList");
+    var li = document.createElement("li");
+    li.appendChild(document.createTextNode(anIngredient));
+    ul.appendChild(li);
+  }
```

Comment:
You did a good job using the useState hooks and state variables here. The code you have here works, there is an easier way however that will save you from having to use DOM manipulation. 
If you had a state variable called ingredientList, and used this to map you ingredients on the page, you could just use setIngredientList to add the new ingredient to the list. This will cause the component to rerender (or you could force it to) and now it would map all the old ingredients and the new one. This will allow you to update lists on screen without DOM manipulation (this addIngredient function). 
What you have works fine so you are good to move on, just something I wanted to point out that could make things easier.